src/natcap/invest/habitat_quality.py

```diff
@@ -278,6 +278,70 @@
                     "created_if": "lulc_bas_path and lulc_fut_path",
                     "bands": {1: {"type": "ratio"}}
                 },
+                "rarity_c.csv": {
+                    "about": ("Table of rarity values by LULC code for the "
+                              "current landscape"),
+                    "index_col": "LULC_code",
```

Comment:
I think we have started to standardize using all lowercase for table outputs. Could we have this be `lulc_code`?